theforeman.org/pipelines/lib/foreman.groovy

```diff
@@ -1,3 +1,16 @@
+def databaseUrlForTask(task) {
+    if (task == 'assets:precompile') {
+        return 'nulldb://nohost'
+    } else {
+        database = UUID.randomUUID().toString()
+        return "postgresql://foreman:foreman@localhost/foreman-${database}"
```

Comment:
somehow all tests end up using the same DB. and that breaks things

Reply:
Is this still true with the changes that moved the function call into the matrix?

Reply:
Tested in https://ci.theforeman.org/blue/organizations/jenkins/ewoud-test/detail/ewoud-test/2/pipeline/77 (likely to be deleted) but there I see:

> [2025-03-06T13:45:30.061Z] Created database 'foreman-c85b58bc-19d6-453b-8e06-baa2f933dae5'
> [2025-03-06T13:44:23.509Z] Created database 'foreman-7d94bab3-8303-425f-96a3-d7d8db402c5b'

So moving it into the matrix helps.